draft-xyz-rats-psa-endorsements.md

```diff
@@ -104,24 +111,44 @@ There are three basic types of PSA endorsements:
 * Certification Claims ({{sec-certificates}}), i.e., metadata that describe
   the certification status associated with a PSA device.
 
+Each type is encoded using a separate CoMID.
+
+## PSA RoT Identification
+{: #sec-impl-id}
+
+Each PSA Endorsement is associated with a well defined immutable PSA RoT.  The
+unique identifier of such PSA RoT, known as the Implementation ID (see Section
+3.2.2 of {{PSA-TOKEN}}), MUST be encoded in the top-level `element-name-map`
+(2) of the CoMID using the `$class-id-type-choice` (2) entry with type
+`tagged-impl-id`, as shown in {{ex-implementation-id}}.  Note that this implies
+that each CoMID is scoped to one and one only PSA RoT or, in other words, that
```

Comment:
On line 124 you mean
to one and only one

Reply:
no, I intentionally wanted to use that form :-)

Reply:
This does not sound right to me, in terms of usage ?